db/migrate/20251204202855_create_ai_transcriptions_table.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAiTranscriptionsTable < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ai_transcriptions do |t|
+      t.references :page, null: false, foreign_key: { on_delete: :cascade }, type: :integer
+      t.text :source_text
+      t.string :model, null: false
```

Comment:
Let's add a string for prompt or prompt filename here -- possibly in the future.